README.md

```diff
@@ -75,6 +75,72 @@ python manage.py test
 flake8 .
 ```
 
+### Windows 10 Users
+ Please install and set up the following packages first, upgrade if you find the package is already installed:
+
+* Download and install [Python3](https://www.python.org/downloads/) and ensure to 'Add path' while installing. Run the 'python3 -V' command to see the version you have installed.
+
+* Download and install [pip](https://pip.pypa.io/en/latest/installing/) and follow the installing with get-pip.py instructions.
+
```

Comment:
How about you word this section like this -> "Download pip <insert appropriate link> and follow the instructions in the link as an installation guide".

Reply:
Pip should be installed in a virtual environment allowing other packages in too.

Reply:
Pip is installed globally, not in a virtualenv. Virtualenvs are for packages which you may not want to install globally.

Reply:
Yeah Delphine, Pip is installed globally and not in a virtualenv. 

Okay if i got you correctly NK, i should adjust only that line and then im good to go right

Reply:
There are other comments you may need to address, but for this one, yes, reword the sentence.

Reply:
Done